File: leetcode/by_level/easy/py/709.py
def Solution(str):
  result = []
  strList = list(str)
  for character in strList:
    codeNum = ord(character)
    if (codeNum >= 65 and codeNum <= 90):
      codeNum += 32
    result.append(chr(codeNum))

  s = ''  
  return s.join(result)

print(Solution('Hello'))

File: leetcode/by_level/easy/js/709.js
const solution = (str) => {
  let list = str.split('')
  let result = []
  for (let i = 0; i < list.length; i++) {
    const item = list[i]
    let charCode = item.charCodeAt()
    if (charCode >= 65 && charCode <= 90) {
      charCode += 32
    }
    console.log(charCode)
    result.push(String.fromCharCode(charCode))
  }
  return result.join('')
}

console.log(solution('Hello'))